src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +238,40 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const format = useCallback(
```

Comment:
How about changing this function name to something like `handleFormatSelection` or `handleFormatSelectedText`? I think the `format` name is too general.

Reply:
That makes sense. Updated